developer/src/kmcmplib/tests/api-test.cpp

```diff
@@ -54,8 +54,11 @@ void setup() {
 
 void test_kmcmp_CompileKeyboardFile() {
   char kmn_file[L_tmpnam], kmx_file[L_tmpnam];
-  tmpnam(kmn_file);
-  tmpnam(kmx_file);
+  strncpy(kmn_file, "test_kmcmp_CompileKeyboardFile.kmn", L_tmpnam);
+  strncpy(kmx_file, "test_kmcmp_CompileKeyboardFile.kmx", L_tmpnam);
+
+  puts(kmn_file);
+  puts(kmx_file);
```

Comment:
Is this because tmpnam is not available? Feels like a step backwards

Reply:
It's marked as deprecated. And warnings-are-errors is on.

Reply:
OK, we'll need to find a way to keep randomized temp file names for those situations where we have multiple processes using same temp